read me edit (#31)
LBHackney-IT/resident-contact-api: README.md

> Resident Contact API will return a list of contact details (inc. mobile, uprn, email) for residents that we have stored.

Reviewer: This feels more like a description for the LIST endpoint rather than the API

Author: I wanted opinions on this, do you think I should remove it, it was already there so wasn't too sure

Reviewer: I think you could rephrase it to describe the API. Maybe something like:
```suggestion
Resident Contact API allows access to saved contact details (inc. mobile, uprn, email) for residents that we have stored.
```

> The contact details can be retrieved through the use of a GET Endpoint which will return all the data or a specific Id too.

Reviewer: ```suggestion
The contact details can be retrieved through the use of two GET Endpoints. One will return all the contact detail records stored, the other will return a single record for specified ID.
```
